docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -0,0 +1,3023 @@
+</tr><tr id="TBL-1-7-"><td class="td11" style="white-space:nowrap; text-align:left;" id="TBL-1-7-1"> <span class="rm-lmr-10x-x-109">E  </span></td><td class="td11" style="white-space:nowrap; text-align:left;" id="TBL-1-7-2"> <span class="rm-lmr-10x-x-109">Bovine Serum Albumin  </span></td><td class="td11" style="white-space:nowrap; text-align:left;" id="TBL-1-7-3"> <i>Bovine Serum</i>         </td><td class="td11" style="white-space:nowrap; text-align:left;" id="TBL-1-7-4"> <span class="rm-lmr-10x-x-109">66 KD                </span></td><td class="td11" style="white-space:nowrap; text-align:right;" id="TBL-1-7-5"> <span class="rm-lmr-10x-x-109">11  </span></td><td class="td11" style="white-space:nowrap; text-align:right;" id="TBL-1-7-6"> <span class="rm-lmr-10x-x-109">0.6  </span></td><td class="td11" style="white-space:nowrap; text-align:right;" id="TBL-1-7-7"> <span class="rm-lmr-10x-x-109">10  </span></td><td class="td11" style="white-space:nowrap; text-align:right;" id="TBL-1-7-8"> <span class="rm-lmr-10x-x-109">500  </span></td>
+</tr><tr id="TBL-1-8-"><td class="td11" style="white-space:nowrap; text-align:left;" id="TBL-1-8-1"> <span class="rm-lmr-10x-x-109">F  </span></td><td class="td11" style="white-space:nowrap; text-align:left;" id="TBL-1-8-2"> <span class="rm-lmr-10x-x-109">Carbonic Anhydrase     </span></td><td class="td11" style="white-space:nowrap; text-align:left;" id="TBL-1-8-3"> <i>Bovine Erythrocytes</i>  </td><td class="td11" style="white-space:nowrap; text-align:left;" id="TBL-1-8-4"> <span class="rm-lmr-10x-x-109">29 KD                </span></td><td class="td11" style="white-space:nowrap; text-align:right;" id="TBL-1-8-5"> <span class="rm-lmr-10x-x-109">10  </span></td><td class="td11" style="white-space:nowrap; text-align:right;" id="TBL-1-8-6"> <span class="rm-lmr-10x-x-109">500  </span></td><td class="td11" style="white-space:nowrap; text-align:right;" id="TBL-1-8-7"> <span class="rm-lmr-10x-x-109">11  </span></td><td class="td11" style="white-space:nowrap; text-align:right;" id="TBL-1-8-8"> <span class="rm-lmr-10x-x-109">0.6  </span></td>
+
+</tr></tbody></table></div>
+
+</figure>
+</div>
+
+#### Identification and quantification
+
+|![KNIME data analysis of iPRG LFQ data](../images/openms-user-tutorial/labelfree/iPRG_lfq.png)|
+|:--:|
+|Figure 18: KNIME data analysis of iPRG LFQ data.|
+
+The iPRG LFQ workflow (Fig. 18) consists of an identification and a quantification part. The identification is achieved by searching the computationally calculated MS2 spectra from a sequence database (**Input File** node, here with the given database from iPRG:
+`ExamplexData` ► `iPRG2015` ► `database` ► `iPRG2015xtargetxdecoyxnocontaminants.fasta`
```

Comment:
The "x" are errors from the latex package. They should all be under scores `_`. please check other occurrences.